docker/swift-cross-platform/download.sh

```
@@ -0,0 +1,9 @@
+#!/bin/sh
+# Download files, from the tarball of the swift page 
+# They are a copy paste from https://www.swift.org/install/linux/ubuntu/24_04/#latest
+echo "Selecting platform for $1"
+if [ $1 == "x64_64" ]; then 
```

Shellcheck (0.10.0)

[warning] In POSIX sh, == in place of = is undefined.

(SC3014)